demo/src/examples/load-media-form-component.js

```diff
@@ -114,12 +115,35 @@ export class LoadMediaFormComponent extends LitElement {
     `;
   }
 
+  updated(_changedProperties) {
+    super.updated(_changedProperties);
+
+    if (_changedProperties.has('drmSettingsShown') && this.drmSettingsShown) {
+      this.shadowRoot.querySelector('.drm-settings-container').classList.toggle('active', true);
```

Comment:
In this case this might be easier to read
```suggestion
      this.shadowRoot.querySelector('.drm-settings-container').classList.add('active');
```